src/core/trulens/core/tru.py

```diff
@@ -379,7 +379,7 @@ def add_record_nowait(
 
     def batch_loop(self):
         while True:
-            time.sleep(self.RECORDS_BATCH_TIMEOUT)
+            time.sleep(self.RECORDS_BATCH_TIMEOUT_IN_SEC)
```

Comment:
@sfc-gh-pdharmana I renamed this to be more explicit

Reply:
Thanks